src/handlers/site_upload/powerset_merge.py

```diff
@@ -13,6 +13,10 @@
 
 from src.handlers.shared import awswrangler_functions, decorators, enums, functions
 
+log_level = os.environ.get("LAMBDA_LOG_LEVEL", "INFO")
+logger = logging.getLogger()
+logger.setLevel(log_level)
+
```

Comment:
I'll centralize this at some point, but right now i'm mostly focused on just getting dashboard development unblocked.